projects/angular/src/accordion/stepper/stepper-panel.html

```diff
@@ -0,0 +1,64 @@
+<ng-container *ngIf="panel | async; let panel">
+  <div [ngClass]="getPanelStateClasses(panel)">
+    <div class="clr-accordion-header clr-stepper-header">
+      <button
+        type="button"
+        class="clr-accordion-header-button"
+        (click)="togglePanel()"
+        [id]="getAccordionHeaderId(panel.templateId)"
+        [disabled]="isAccordion && panel.disabled"
+        [attr.aria-disabled]="!isAccordion && panel.disabled"
```

Comment:
The `isAccordion` properties and logic is no longer needed since the accordion and stepper panels no longer share a template. See 3c442af38a7169ce8a33cac1f0b851f64ab9e0fd.

Reply:
I completly agree, but it is a `public` property. 
Removing it will be considered as API change.

Reply:
It is only `public` so that it could be used in the template. It is not a documented property that anyone should be using. I think we should remove it.

Reply:
Corrected in commit: 08eed4a